agentic_app_manager_next_25/ios/Podfile

```diff
@@ -1,5 +1,5 @@
 # Uncomment this line to define a global platform for your project
-# platform :ios, '12.0'
+platform :ios, '13.0'
```

Comment:
You'll need to update `macos/Podfile` as well. Or, just delete the `macos` Runner.